src/hooks/useConnectWallet.ts

```diff
@@ -77,15 +95,38 @@ export const useConnectWallet = (): ConnectWalletAction => {
     }
     connect({ connector })
   }
-
   const handleSignOutViaWalletConnect = async () => {
     await disconnect()
   }
 
+  /**
+   * Solana:
+   * Down below are Solana Wallet handlers and actions
+   */
+  const { setVisible } = useWalletModal()
+  const { publicKey: solanaPublicKey, disconnect: disconnectSolana } =
```

Comment:
Just `solanaWallet` (`solanaWallet.publicKey` `solanaWallet.disconnect`) would be more readable. The same for other hooks above.